charts/connect/CHANGELOG.md

```diff
@@ -3,14 +3,22 @@
 
 ## Features
 * A user-friendly description of a new feature. {issue-number}
-* Add priorityClassName values to operator and connect. {#201}
 
 ## Fixes
 * A user-friendly description of a fix. {issue-number}
 
 ## Security
 * A user-friendly description of a security fix. {issue-number}
 
+[//]: # (START/v2.0.0)
+# v2.0.0
+
+## Features
+* Add support for configuring the operator to use a 1Password Service Account. {#226}
+
+## Security
+* Default Connect service type changed to ClusterIP. (#194)
```

Comment:
I think it would make more sense to label this as a breaking change than as a security fix.